src/features/settings/Settings.tsx

```diff
@@ -396,7 +397,7 @@ const Settings = () => {
       },
     ]
 
-    if (Config.SOLANA_PREVIEW === 'true') {
+    if (betaAccess?.publicKeys.includes(currentAccount?.address || '')) {
```

Comment:
Should we all a null guard for the .includes?

Reply:
Will do